Mathlib/Tactic/extractGoal.lean

```diff
@@ -0,0 +1,80 @@
+/-
+Copyright (c) 2017 Simon Hudon. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Simon Hudon, Damiano Testa
+-/
+import Std.Data.List.Basic
+
+/-!
+#  `extractGoal`: Format the current goal as a stand-alone example
```

Comment:
The tactic syntax should probably be ` extract_goal` though

Reply:
Also, changed!